src/Toolkit/Toolkit.iOS/UI/Controls/PopupViewer/PopupViewerTableSource.cs

```diff
@@ -60,7 +60,8 @@ public override UITableViewCell GetCell(UITableView tableView, NSIndexPath index
             var cell = tableView.DequeueReusableCell(CellId) as DetailsItemCell;
             if (cell == null)
             {
-                return null;
+                System.Diagnostics.Debug.Assert(false, "Cell should not be null");
+                return null!;
```

Comment:
Should we just throw here? Might be more helpful than causing a NullReferenceException later.